src/__tests__/auth/socialAuth.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import {  shallow } from "enzyme";
+import {SocialAuth} from "../../components/auth/socialAuth";
+import userReducer from "../../reducers/authReducers/socialAuthReducer";
+import { SOCIAL_AUTH, LOGIN } from "../../actions/actionTypes";
+
+describe("socialAuth component", () => {
+  const wrapper = shallow(<SocialAuth/>);
+  it("it has a div with class social-btn text-center ", () => {
+    expect(wrapper.find("div").hasClass("social-btn text-center")).toBe(true);
+  
+  });
+ 
+});
+
+describe("socialAuthReducer", () => {
+  it("it has a default state", () => {
```

Comment:
Remove the 'it' in the test description as its already written by default

Reply:
@LehruAngela  I have made this change